core/controllers/library.py

```diff
@@ -299,8 +299,7 @@ def get(self):
         if include_private_exps:
             summaries = (
                 summary_services.get_displayable_exp_summary_dicts_matching_ids(
-                    exp_ids,
-                    editor_user_id=editor_user_id))
+                    exp_ids, self.user))
```

Comment:
If an arg has a default value, include the name of the arg explicitly (`user=self.user`).